compiler/natives/src/testing/example.go

```diff
@@ -0,0 +1,70 @@
+// +build js
+
+package testing
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+func runExample(eg InternalExample) (ok bool) {
+	if *chatty {
+		fmt.Printf("=== RUN   %s\n", eg.Name)
+	}
+
+	// Capture stdout.
+	stdout := os.Stdout
+	w, err := tempFile("." + eg.Name + ".stdout.")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+	os.Stdout = w
+
+	start := time.Now()
+	ok = true
+
+	// Clean up in a deferred call so we can recover if the example panics.
+	defer func() {
+		dstr := fmtDuration(time.Now().Sub(start))
+
+		// Close pipe, restore stdout, get output.
```

Comment:
Minor, we've changed the code to use file rather than pipe, but the comment still says pipe.

I can fix it.